tests/languages/docker_test.py

```diff
@@ -12,3 +16,75 @@ def invalid_attribute():
         getgid=invalid_attribute,
     ):
         assert docker.get_docker_user() == ()
+
+
+class TestInDocker:
```

Comment:
we don't use test classes in this project

Reply:
Well, I do want to group tests for a given function together. Test classes are the best suited for this. Having bunch of files like `test_modulename_functionname` is a bit too much IMO

Reply:
we don't use test classes in this project

Reply:
Please be more constructive. I would like to have tests grouped together by their test target. Test classes is the built-in way to do so and in comparison with bare functions it improves both readability of the code an readability of reports. They don't introduce any extra tremendous boilerplate, so what's the problem?